### DISCUSSION TITLE
Issue with Vue DevTools Beta Extension on Microsoft Edge

### DISCUSSION THREAD

**Discussion by webfansplz** [microsoft/MicrosoftEdge-Extensions#166] [Q&A]
Hello,

I am the creator of [Vue DevTools](https://github.com/vuejs/devtools-next) project. We recently released a new version of the devtools, vue-devtools-beta v7.0.0-beta.2, to the Chrome Web Store. The installation on Chrome proceeds without any issues, but we are encountering a problem when trying to install the same extension on Microsoft Edge. Users are receiving the following error message:

```shell
“This extension contains malware.”
```

We have thoroughly checked our code and ensured that it complies with all security guidelines. However, we are unable to replicate the issue on Chrome and are uncertain about the cause of this warning on Edge.

Could you please assist us in identifying the source of this problem and provide guidance on how to resolve it? Any insights into why Edge is flagging the extension as containing malware would be greatly appreciated.
```
Steps to Reproduce:

1.	Open Microsoft Edge.
2.	Go to the Chrome Web Store and search for vue-devtools-beta v7.0.0-beta.2.
3.	Attempt to install the extension.
4.	Observe the error message stating that the extension contains malware.
```
Expected Behavior:

The extension should install without any errors, similar to the installation process on Google Chrome.

![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/22515951/7649e3fe-77df-41f1-bd63-4f4a86b7654c)

**Comment by ManikanthMSFT** [comment]
Hi @webfansplz, we’ve noticed that your query has been successfully resolved, as confirmed by your recent email. If you have any further concerns or require additional assistance, please don’t hesitate to contact us. We’re here to help.
